JS/Lesson32/onder/my-app/src/components/RecipeDetail/index.jsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
```

Comment:
NIce to have: add styles to style the page nicely:

```suggestion
import './styles.css';
```

CSS code:
```
.recipe-details-container {
  display: grid;
  grid-template-rows: 50px 1fr;
  grid-template-columns: 1fr 1fr;
  grid-template-areas: "heading heading"
  "image-content text-content";
  gap: 20px;
}

.heading {
  grid-area: heading;
}

.image-content {
  grid-area: image-content;
}

.text-content {
  grid-area: text-content;
}

.recipe-thumbnail {
  width: 100%;
}
```